sh/fish/tabry_fish.fish

```diff
@@ -21,7 +21,9 @@ function __fish_tabry_internal_invoke
   set cmd (commandline)
 
   set result ($TABRY_EXECUTABLE $TABRY_ARG "$cmd" "$cursor_position")
-  echo $result
+  for arg in $result
+    echo $arg
```

Comment:
Hmm OK... In bash, and it seems like in fish, the -n flag causes to leave off a newline. In a command "echo -n", "-n" is interpreted as a flag. printf is a more dependable way of printing arbitrary strings ("echo -- -n" also seems to work to print out "-n").
![image](https://github.com/evanbattaglia/tabry/assets/669236/9f78ae59-2dad-491f-9e17-7baf085b6f0d)

In your screenshot, is it printing out "-n\tCount of things in list" (I mean, with an actual tab character)? If you have a flag "-n" with no description does it error, or if you have "-n" as an option (e.g., under an arg or flagarg block: opts const "-n")

Reply:
It does print out `-n\tCount of things in list`:

```
> set -x TABRY_FISH_MODE 1
> /app/examples/all_in_one/hello completion "hello things list " 18
-n	Count of things in list
-a
-b
```
 
<img width="1119" alt="Screen Shot 2023-07-18 at 8 58 34 PM" src="https://github.com/evanbattaglia/tabry/assets/4752550/ab84edae-5cfa-46eb-9628-7a3f6d621fca">
I'm honestly not sure why it doesn't matter here but not in other places.

It definitely _does_ matter in some cases, like here: https://github.com/evanbattaglia/tabry/blob/master/sh/fish/tabry_fish.fish#L51, where I need to add a `--` to `string match` to escape any strings that start with `-`

Reply:
I just tried to make sure with some flags like -a, -e, -n ... it seems like it works, I think since you are actually printing out "-e\t", "-n\t" in those cases. "echo -a" always works -- echo prints out strings unless it interprets them as _supported_ flags, -e and -n are two examples of supported flags. So
```
echo -e
```
prints nothing, but
```
echo -e\t`
```
prints "-e" followed by a tab. I still think using printf is a more dependable method, but as long as you're sure you'll always be writing that tab character, it's fine.